examples/PyTorch/Inference/BERT/README.md

```diff
@@ -0,0 +1,45 @@
+# DISC optimization example for PyTorch BERT-large model.
+
+This repository provides a script showing how to optimize a BERT large inference
+model for PyTorch with DISC. The BERT-large model is constructed with Hugging
+Face API, with fake parameter values. It prepares fake input data for execution.
+It also shows a performance comparison between BladeDISC and TensorRT.
+
+
+## Configure and run.
+
+These python packages are required for executing this example:
+
+- torch >= 1.6.0
+- transformers
+- torch_blade
+- volksdep
+
+To build and install `torch_blade` package, please refer to
+["Installation of TorchBlade"](../build_from_source.md) and
```

Comment:
> ["Installation of TorchBlade"](../build_from_source.md) and
["Install BladeDISC With Docker"](../install_with_docker.md).

no `build_from_source.md` file in the parent dir?

"Installation of TorchBlade" and "Install BladeDISC with Docker"
=> "Build BladeDISC from source" or "Install BladeDISC with Docker"

Reply:
Fixed. Thanks.